engine/test/integration_tests/dm_lightning_checkpoint/run.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -eu
+
+WORK_DIR=$OUT_DIR/$TEST_NAME
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+
+CONFIG="$DOCKER_COMPOSE_DIR/3m3e_with_s3.yaml $DOCKER_COMPOSE_DIR/dm_databases.yaml"
+CONFIG=$(adjust_config $OUT_DIR $TEST_NAME $CONFIG)
+echo "using adjusted configs to deploy cluster: $CONFIG"
+
+function run() {
+	start_engine_cluster $CONFIG
+	wait_mysql_online.sh --port 3306
+	wait_mysql_online.sh --port 3307
+	wait_mysql_online.sh --port 4000
+
+	# prepare MySQL global variables and data
+	run_sql_file $CUR_DIR/data/db1.prepare.sql
+
+	# create job & wait for job finished
```

Comment:
if there a test that two lightning are running simultaneously?

Reply:
I will add two jobs in this test.

Reply:
Fix it.